measure_extinction/extdata.py

```diff
@@ -995,15 +999,24 @@ def plot(
             if "AV" not in self.columns.keys():
                 self.trans_elv_alav()
             av = _get_column_val(self.columns["AV"])
-            if self.type_rel_band != "V":  # not sure if this works (where is RV given?)
-                # use F04 model to convert AV to AX
-                rv = _get_column_val(self.columns["RV"])
-                emod = F04(rv)
-                (indx,) = np.where(self.type_rel_band == self.names["BAND"])
-                axav = emod(self.waves["BAND"][indx[0]])
+            if self.type_rel_band != "V":
+                if self.type_rel_band in self.names["BAND"]:
+                    (bindx,) = np.where(self.names["BAND"] == self.type_rel_band)
+                    ax = self.exts["BAND"][bindx[0]] + av
```

Comment:
I don't think this is calculation correct. self.exts is already in "alav" units at this point (see line 1000)...